src/overlays/actors/ovl_Bg_Goron_Oyu/z_bg_goron_oyu.h

```diff
@@ -4,12 +4,21 @@
 #include <global.h>
 
 struct BgGoronOyu;
-
+typedef void (*BgGoronOyuActionFunc)(struct BgGoronOyu*, GlobalContext*);
 typedef struct BgGoronOyu {
-    /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x3C];
-} BgGoronOyu; // size = 0x180
+    /* 0x000 */ DynaPolyActor polyActor;
+    /* 0x15C */ BgGoronOyuActionFunc actionFunc;
+    /* 0x160 */ u8 gap160[4];
+    /* 0x164 */ f32 flt164;
+    /* 0x168 */ Vec3f collisionHeaderMinVec3f;
+    /* 0x174 */ f32 collisionHeaderMaxX;
+    /* 0x178 */ f32 collisionHeaderMaxY;
+    /* 0x17C */ s16 initialActorCutscene;
+    /* 0x17E */ u16 unk17E;
+} BgGoronOyu; // sizeof=0x180, align=0x4
```

Comment:
```suggestion
} BgGoronOyu; // size = 0x180
```
I don't think the align is necessary for actors